SUMMARY.md

```diff
@@ -10,23 +10,19 @@
    * [Input Data](input_data.md)
    * [Simulation, Computation](simulation.md)
    * [Reforms](reforms.md)
-* [Getting Started](getting-started.md)
-   * [First Steps](first_step.md)
-* [Thinking in vectors](thinking-in-vectors.md)
-* [Publishing results](publishing.md)
-* [Coding the legislation](coding-the-legislation/README.md)
-   * [Coding a formula: basic example](coding-the-legislation/10_basic_example.md)
-   * [Introducing an input variable](coding-the-legislation/20_input_variables.md)
-   * [Vector calculus limitations](coding-the-legislation/25_vectorial_computing.md)
-   * [Case disjunction](coding-the-legislation/30_case_disjunction.md)
-   * [Periods](coding-the-legislation/35_periods.md)
-   * [Legislation evolutions](coding-the-legislation/40_legislation_evolutions.md)
-   * [Entities](coding-the-legislation/50_entities.md)
-   * [Reforms](coding-the-legislation/reforms.md)
-   * [Inferences](coding-the-legislation/inferences.md)
-   * [Writing YAML tests](coding-the-legislation/writing_yaml_tests.md)
-   * [Parameters](coding-the-legislation/legislation_parameters.md)
-   * [Bootstrapping a new country package](coding-the-legislation/bootstrapping_a_new_country_package.md)
+* [From law to code](from-law-to-code/README.md)
```

Comment:
Renaming the directory `coding-the-legislation` has a lot of impacts ! Did you check _all_ the other repositories for links pointing to one of these pages ? 

I'd feel much more confortable with changing the title without changing the URL.